packages/orbit-components/src/InputFile/index.tsx

```diff
@@ -157,7 +157,6 @@ const InputFile = React.forwardRef<HTMLDivElement, Props>((props, ref) => {
           help={help}
           error={error}
           helpClosable={helpClosable}
-          inputSize="normal"
```

Comment:
This is not a breaking change. If anything, this change can be included in the same commit as the BC for ErrorFormTooltip. But removing const props used internally is not a BC